pom.xml

```
@@ -95,54 +93,229 @@
+                <module>modules/org.restlet.ext.gson</module>
+                <module>modules/org.restlet.ext.inject</module>
+                <module>modules/org.restlet.ext.html</module>
+                <module>modules/org.restlet.ext.httpclient</module>
+                <module>modules/org.restlet.ext.jaas</module>
+                <module>modules/org.restlet.ext.jackson</module>
+                <module>modules/org.restlet.ext.jaxb</module>
+                <module>modules/org.restlet.ext.jetty</module>
+                <module>modules/org.restlet.ext.json</module>
+                <module>modules/org.restlet.ext.servlet</module>
+                <module>modules/org.restlet.ext.slf4j</module>
+                <module>modules/org.restlet.ext.spring</module>
+                <module>modules/org.restlet.ext.thymeleaf</module>
+                <module>modules/org.restlet.ext.velocity</module>
+                <module>modules/org.restlet.ext.xml</module>
+            </modules>
+        </profile>
         <profile>
             <id>test</id>
             <modules>
+                <module>modules/org.restlet</module>
+                <module>modules/org.restlet.example</module>
+                <module>modules/org.restlet.ext.crypto</module>
+                <module>modules/org.restlet.ext.fileupload</module>
+                <module>modules/org.restlet.ext.freemarker</module>
+                <module>modules/org.restlet.ext.gson</module>
+                <module>modules/org.restlet.ext.inject</module>
+                <module>modules/org.restlet.ext.html</module>
+                <module>modules/org.restlet.ext.httpclient</module>
+                <module>modules/org.restlet.ext.jaas</module>
+                <module>modules/org.restlet.ext.jackson</module>
+                <module>modules/org.restlet.ext.jaxb</module>
+                <module>modules/org.restlet.ext.jetty</module>
+                <module>modules/org.restlet.ext.json</module>
+                <module>modules/org.restlet.ext.servlet</module>
+                <module>modules/org.restlet.ext.slf4j</module>
+                <module>modules/org.restlet.ext.spring</module>
+                <module>modules/org.restlet.ext.thymeleaf</module>
+                <module>modules/org.restlet.ext.velocity</module>
+                <module>modules/org.restlet.ext.xml</module>
                 <module>modules/org.restlet.test</module>
             </modules>
         </profile>
         <profile>
-            <id>check-cve</id>
+            <id>github</id>
+            <activation>
+                <property>
+                    <name>publish</name>
+                    <value>github</value>
+                </property>
+            </activation>
+            <distributionManagement>
+                <repository>
+                    <id>github</id>
+                    <name>GitHub Packages</name>
+                    <url>https://maven.pkg.github.com/xSAVIKx/restlet-framework-java</url>
```

Should be `restlet`?

```suggestion
                    <url>https://maven.pkg.github.com/restlet/restlet-framework-java</url>
```